packages/std/src/proto/DESMOS_COMMIT

```diff
@@ -0,0 +1 @@
+581e029ce1c0f84413a961258b4d2048ab386496
```

Comment:
Do we need this file? Its seem unused.

Reply:
This file is for easily checking the current generated std library from which Desmos version.